samples/benchmarks/coremark/README.rst

```diff
@@ -173,6 +189,11 @@ Testing
 After programming the sample to your development kit, complete the following steps to test it:
 
 1. |connect_terminal|
+
+   .. note::
+      To see all logging information in case of the multi-core board targets and the standard logging mode, you must open a terminal for each active core.
```

Comment:
`the multi-core board targets with the standard logging mode` - "and" could mean that in both cases we should use 2 terminals.

Reply:
`To see all logging information in the case of the multi-core board targets that use the standard logging mode`

Is it better?